### PR TITLE
Django 111

### PR DESCRIPTION
**Pull request by somexpert**
Add compatibility primarily for Django 1.11

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko

File: tests/settings/base.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import os
-local_path = lambda path: os.path.join(os.path.dirname(__file__), path)
+local_path = lambda path: os.path.join(os.path.dirname(__file__), path)  # noqa
```
Comment (ldko) [inline tests/settings/base.py]: How is `local_path` used? Is it used?

File: tests/settings/test_mysql.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
         'NAME': 'name',
-        'USER': os.getenv('DB_MYSQL_USER', default="root"),
-        'PASSWORD': os.getenv('DB_PASSWORD', default="root"),
-        'HOST': os.getenv('DB_HOST', default='mysql'),
+        'USER': os.getenv('DB_MYSQL_USER', default="root"),  # noqa
+        'PASSWORD': os.getenv('DB_PASSWORD', default="root"),  # noqa
```
Comment (ldko) [inline tests/settings/test_mysql.py]: We could use single quotes around `root` for consistency.

**Review by ldko — COMMENTED** [review]
Works for me, just a couple very minor comments

File: tests/settings/base.py (hunk 1 above)
Comment (somexpert) [reply tests/settings/base.py]: Ah, doesn't look like it's used at all. I'll get rid of it.

File: tests/settings/test_mysql.py (hunk 2 above)
Comment (somexpert) [reply tests/settings/test_mysql.py]: will do!

**Comment by ldko** [comment]
:+1:

**Review by ldko — APPROVED** [review]

**Pull request merged**